editor/src/components/canvas/commands/push-intended-bounds-and-update-groups-command.ts

```diff
@@ -242,25 +242,34 @@ function getUpdateResizedGroupChildrenCommands(
   let commandsToRun: Array<CanvasCommand> = []
   let updatedElements: Array<ElementPath> = []
 
-  Object.entries(updatedLocalFrames).forEach(([pathStr, frameAndTarget]) => {
-    const elementToUpdate = EP.fromString(pathStr)
-    const metadata = MetadataUtils.findElementByElementPath(editor.jsxMetadata, elementToUpdate)
+  Object.entries(updatedLocalFrames)
+    .filter(([path]) => {
```

Comment:
Could fold this into the `forEach` to avoid creating an intermediary array.

Reply:
done here https://github.com/concrete-utopia/utopia/pull/4080/commits/546100ee93205cb9587613aa33a093e565292af1